labelbox/schema/batch.py

```diff
@@ -174,3 +180,7 @@ def delete_labels(self, set_labels_as_template=False) -> None:
                 },
             experimental=True)
         return res
+
+    @property
+    def failed_data_row_ids(self):
+        return self._failed_data_row_ids
```

Comment:
Is this a property so that in the future we can update the logic for how this is populated?

Reply:
That also works, but first of all to indicate that this is something that is read-only